src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -1,21 +1,24 @@
 package edu.kis.vh.nursery;
 
-public class FIFORhymer extends defaultCountingOutRhymer {
+public class FIFORhymer extends DefaultCountingOutRhymer {
+
+	private DefaultCountingOutRhymer temp = new DefaultCountingOutRhymer();
 
-	public defaultCountingOutRhymer temp = new defaultCountingOutRhymer();
 	
 	@Override
 	public int countOut() {
 		while (!callCheck())
-			
-		temp.countIn(super.countOut());
+			getTemp().countIn(super.countOut());
 		
-		int ret = temp.countOut();
+		int ret = getTemp().countOut();
```

Comment:
Shouldn't be encapsulated (typical private field). This is an exaggeration. Using private fields internally usually do not require encapsulation.